docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -260,6 +236,9 @@ This list of ciphers is provided as an example only. Actual ciphers should be cu
 The list of supported ciphers should be constructed according to the TLS supported versions.
 Ensure that the cipher list has matches with non-AT-TLS-aware clients.
 
+<details>
+<summary>Click here for an example of Cipher parameters.</summary>
```

Comment:
We should mentioned, that the list should be verified by security to stuff to match with their policy. The list is just example, it is not suggested one.

Reply:
I can add a note, but we are already saying this is just an example. We are not recommending the list.

Reply:
This is mentioned in the main section where we explain the ciphers